.storybook/docs-root.css

```diff
@@ -156,6 +156,16 @@
   border-radius: 5px 5px 0px 0px !important;
 }
 
+/* Reduce font size of CodeSandbox and Show Code button when zoomed or small window width*/
+/* https://github.com/microsoft/fluentui/issues/22764 */
+
+@media screen and (max-width: 380px) {
```

Comment:
what does 380px mean?

Reply:
380 was a safe number to reduce the font size for small screens, but not reduce until it was needed. If we change button styles/size/text we'd need to look at the number again